.gitmodules

```diff
@@ -18,7 +18,7 @@
 	path = deps/juno
 	url = https://github.com/CosmosContracts/juno.git
 [submodule "deps/osmosis"]
-	# Commit: v16.1.1-no-fees
+	# Commit: v20.5.0-no-fees
```

Comment:
shouldn't we use the latest mainnet version? I assume v20.5 has the patch for this

Reply:
Yeah normally I would think so, but I bet everything we care about would be sufficiently up to date in v20.2.0 and we don't know if they have the patch in yet.  I doubt this is the main cause of the problem but if we do see it dropping our incoming packets completely this might be something to try.